contributor/04-mechanics-of-contributing-article.asciidoc

```diff
@@ -118,34 +118,34 @@ This might actually be pretty similar to what you may already be doing on your o
 
 ==== Documentation and reviewability
 
-You'll want to ensure that your pull request includes any documentation updates that are relevant to your changes.
+You want to ensure that your pull request includes any documentation updates that are relevant to your changes.
 Should the documentation live in a different place, make sure you add them there and link to them in your pull request.
 
 To make the actual code review as easy as possible for the trusted committer or other persons reviewing it, try to follow these hints:
 
-* Be sure that your pull request includes just the relevant changes for the issue you're completing.
+* Be sure that your pull request includes only the relevant changes for the issue you're completing.
 * Try to avoid super-large commits, commits with unclear commit messages, gazillions of files, incoherent changes (e.g. touching multiple topics).
 * Provide a clear description of what this pull request changes, why it does so, and which issue and design documents (if there were any) it refers to.
 * If there is anything uncommon or unexpected in the pull request, highlight it and provide the explanation. This will make it easier to reason about and solve potential blocking questions that a reviewer might have during the review.
- ** The same goes for scenario where you were unsure of the implementation or your approach - highlight it and ask for an insight.
+ ** The same goes for the scenario where you were unsure of the implementation or your approach - highlight it and ask for an insight.
  ** Be civil and expect civility from the https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/01-introduction.asciidoc[_Trusted Committer's_] review.
 * Making pull requests too broad and large makes them more difficult to review, so it will take much longer before they're accepted.
- ** If you have a larger feature that you are contributing, it often helps to split it in multiple pull requests that get submitted, reviewed and accepted sequentially.
+ ** If you have a larger feature that you are contributing, it often helps to split it in multiple pull requests that are submitted, reviewed and accepted sequentially.
 You can still bind them together with an issue that you are referring to.
-  *** Some tools also have Draft / WIP pull request functionality that you can use to explicitly mark unfinished and non-polished work and still get early feedback from your host team's https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/01-introduction.asciidoc[_Trusted Committers_].
+  *** Some tools also have Draft / WIP pull request functionality that you can use to explicitly mark unfinished and non-polished work and still get early feedback from your host team's https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/02-ensuring-product-quality.asciidoc[_Trusted Committers_].
```

Comment:
What's the motivation of changing this link target? To me this looked like it would address the TC role as a whole, hence linking to the start of the section would make sense to me.

Reply:
I had gone back and forth between keeping the links pointing to the same place, and providing content relevant reference points. 
I remember @rrrutledge suggesting we link to specific articles vs always to the intro, but I could never find a passage specific enough to warrant pointing to a non-intro article.
This portion talks about the TCs in light of their responsibilities towards contributors, and I felt like it is an opportunity to link to the relevant article. 
With that, I also agree that this is not obvious for someone to click on after they clicked on other TC links, so happy to revert if we all decide towards consistency.

Reply:
Likely you'd have to do an [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) to remove the commit affecting the conclusion.
I think trying to merge the other pull request into this one before merging it might be a route that _could_ work using the web UI.